cmd/snap-preseed/reset.go

```diff
@@ -61,7 +62,7 @@ func resetPreseededChroot(preseedChroot string) error {
 	globs = []string{
 		filepath.Join(dirs.SnapDataDir, "*"),
 		filepath.Join(dirs.SnapCacheDir, "*"),
-		filepath.Join(dirs.AppArmorCacheDir, "*"),
+		filepath.Join(apparmor_sandbox.CacheDir, "*"),
```

Comment:
Looks like it's the only other place where this directory is used outside of `interfaces/apparmor` package. Maybe it should actually reside inside that package, but I not strongly set on that.

Reply:
yes, I wasn't quite sure which is the better fit interfaces/apparmor or sandbox/apparmor, I figured sandbox because it's a bit more generic and is used in places like here, so I have a slight preference for keeping it in sandbox

Reply:
those directories have system related paths, not something with snapd? in them, so sandbox seems appropriate